docs/develop.md

```diff
@@ -1,18 +1,426 @@
+For more information, see the
+[contributor-covenant](https://www.contributor-covenant.org/version/2/1/code_of_conduct/).
+```
 
-## docs:
-See https://jupyterbook.org/en/stable/reference/cheatsheet.html#links
+## Collaborative workflow
 
-## Making a new release checklist
-- Zenodo
-- Pypi
-- Black and ...
-- Documentation page
+Collaborative development of WAM2layers takes place on GitHub. You will need a
+[GitHub account](https://github.com/join).
+
+The process is roughly as follows:
+
+1. You encounter a problem or question, or want to suggest an improvement or new feature.
+1. You [Open(s) an
+   issue](https://github.com/WAM2layers/WAM2layers/issues/new/choose) to discuss
+   your idea.
+1. Other developers comment on the issue and together you decide on the best way
+   forward.
+1. If the discussion results in a plan to change the code, you (or someone else)
+   go ahead and make the necessary changes on a local copy of the code.
+1. While you're working, you can create a draft pull requests (PR).
+1. When you are happy with the changes, you mark your PR as "ready for review".
+1. Every pull requests is reviewed by at least one core developer.
+1. When all review comments are addressed and checks pass, the PR is merged.
+1. At this point, the new code is part of the repository, but it is still "unreleased".
+1. The core developers decide when to make a new release.
+1. When the code is released, a new version incorporing your updates appears on
+   Zenodo and PyPI.
+
+### Forking the repository
+
+Depending on your situation, you may have to fork the repository first. Forking
+means you make a duplicate of the repository under your own GitHub account. The
+advantage is that you will be able make changes to your fork without special
+permissions. The disadvantage is that you have to keep your fork in sync with
+the "upstream" repository, i.e. the original WAM2layers repo. Whenever the code
+is updated there, your fork does not automatically get those updates too.
+
+An alternative to forking a repository is to add you to the contributor team of
+WAM2layers. This will give you the rights to push to the original repo. If you
+feel this would make sense for your situation, talk to us.
+
+You can make a fork by navigating to the [original
+repo](https://github.com/WAM2layers/WAM2layers) and pressing the "Fork" button
+(near the right top of the page).
+
+### Cloning the repo (with good authentication)
+
+Cloning a repo means obtaining a local copy. The version on GitHub is referred
+to as the "remote" copy. Navigate to (your fork of) the WAM2layers repo and find
+the big green button that says "code". Now, there are a few options.
+
+Cloning with HTTPS works out of the box. However, whenever you want to "push"
+changes back to the remote, you'll need to authenticate. This can be annoying.
+
+Cloning with SSH is very convenient, but you have to [set up an SSH key
```

Comment:
an SSH key --> a SSH key

Reply:
I think it's actually correct like this, no? Or is this one of those British/American things?

> The general rule for indefinite articles is to use a before consonants and an before vowels.

source: https://blog.apastyle.org/apastyle/2012/04/using-a-or-an-with-acronyms-and-abbreviations.html